pkg/webhook/utils/in_capsule_group.go

```diff
@@ -47,7 +47,12 @@ func (h handler) isCapsuleUser(req admission.Request) bool {
 	if groupList.Find("system:serviceaccounts:kube-system") {
 		return false
 	}
-	return groupList.Find(h.capsuleGroup)
+	for _, group := range h.capsuleGroup {
```

Comment:
It's not optimal here, were ending with `m x O(log n)` here, but at the same time I think it's a pay-off: if you can map users in a single group it works at its best, otherwise prepare for sole bad performances.